### PR TITLE
Wrapping the SSS into a template

### PR DESCRIPTION
**Pull request by apahim**

### CODE REVIEW

File: mso/tempalte.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: selectorsyncset-template
```
Comment (sandhyadalavi) [inline mso/tempalte.yaml]: file name spelled wrong tempalte -> template. can we name file selectorsyncset-template?

Comment (sandhyadalavi) [reply mso/tempalte.yaml]: let me know @apahim and I can do that...

Comment (apahim) [reply mso/tempalte.yaml]: fixed

**Pull request merged**